### ISSUE TITLE
How to set different seed for DDP data sampler for every epoch

### ISSUE THREAD

**Issue by Mountchicken** [huggingface/accelerate#2010]
Hello there!
I am using the following code to build my data loader.
```python
 data_loader_train = DataLoader(
        dataset_train,
        collate_fn=collate_fn,
        batch_size=cfg.data.train_batch_size,
        num_workers=cfg.data.num_workers,
        pin_memory=cfg.data.pin_memory,
    )
data_loader_train = accelerator.prepare(data_loader_train)
```
I am using DDP for training and I want to set different data sample seed for every epoch, so that different epochs will have different batch data orders. How can I do that?

**Comment by BenjaminBossan** [comment]
Maybe I misunderstand your intent, but wouldn't it be sufficient to pass `shuffle=True` to the `DataLoader`?
